### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by mathiascode**
Use https whenever possible, changed some outdated links.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -94,7 +94,7 @@ else()
 endif()
 
 # We need C++11 features, Visual Studio has those from VS2012, but it needs a new platform toolset for those; VS2013 supports them natively:
-# Adapted from http://binglongx.wordpress.com/2013/06/28/set-non-default-platform-toolset-in-cmake/
+# Adapted from https://binglongx.wordpress.com/2013/06/28/set-non-default-platform-toolset-in-cmake/
```
Comment (bearbin) [inline CMakeLists.txt]: This URL no longer exists.

File: GETTING-STARTED.md [hunk 2]
```diff
@@ -3,12 +3,12 @@ Hello! Thanks for wanting to work on this project :smile:, and I hope that this
 Note that this document is about contributing code for Cuberite.
 
  - If you are looking for usage instructions, see the [user manual](https://book.cuberite.org/) instead.
```
Comment (bearbin) [inline GETTING-STARTED.md]: Could you change 'user manual' to 'user's manual' here, as that is the preferred name.

File: src/OSSupport/WinStackWalker.cpp [hunk 3]
```diff
@@ -504,7 +504,7 @@ class StackWalkerInternal
 		pGMI = (tGMI)GetProcAddress(hPsapi, "GetModuleInformation");
 		if ((pEPM == nullptr) || (pGMFNE == nullptr) || (pGMBN == nullptr) || (pGMI == nullptr))
 		{
-			// we couldn�t find all functions
+			// we couldn\B4t find all functions
```
Comment (bearbin) [inline src/OSSupport/WinStackWalker.cpp]: What is this change from?

Comment (mathiascode) [inline src/OSSupport/WinStackWalker.cpp]: Seems like my editor didn't like the file. I'll fix it.

**Comment by madmaxoft** [comment]
I'm on the other side, I prefer links that GitHub web displays inline - after all almost everyone views these files online. But I guess I can live with the separate ones.

**Comment by bearbin** [comment]
@madmaxoft Separate links with markdown should display inline on the web view - it's only for plain text viewing where they display at the end.

**Comment by bearbin** [comment]
Sorry, I think I didn't communicate clearly enough, by separate links I meant these: https://github.github.com/gfm/#reference-link 

They still display inline online, and save space for plain text viewing.

**Comment by mathiascode** [comment]
Like this?
```
Subscribe to [the newsletter] for important updates and project news.
[the newsletter]: https://cuberite.org/news/#subscribe
```

**Comment by mathiascode** [comment]
Like this?
```
Subscribe to [the newsletter][1] for important updates and project news.

[1]: https://cuberite.org/news/#subscribe
```

**Comment by bearbin** [comment]
Yes, like that.

**Comment by bearbin** [comment]
No, it needs a space in front of the [1].

Also, the general best practice is for the second part of the link to be at the end of short documents, and at the end of the major section for long documents.

**Comment by madmaxoft** [comment]
Okay, that looks like a good format.

**Comment by mathiascode** [comment]
Did I miss anything?

File: COMPILING.md [hunk 4]
```diff
@@ -17,15 +17,15 @@ We use Microsoft Visual Studio for Windows compilation. It is possible to use ot
 
 You can find download links for VS2013 Express here: https://go.microsoft.com/?linkid=9832280
```
Comment (bearbin) [inline COMPILING.md]: Missed a link here.

**Pull request merged**